packages/flutter/test/widgets/list_view_test.dart

```diff
@@ -225,11 +316,11 @@ void main() {
 
   testWidgets('didFinishLayout has correct indices', (WidgetTester tester) async {
     final TestSliverChildListDelegate delegate = new TestSliverChildListDelegate(
-      new List<Widget>.generate(20, (int i) {
-        return new Container(
-          child: new Text('$i', textDirection: TextDirection.ltr),
-        );
-      })
+        new List<Widget>.generate(20, (int i) {
+          return new Container(
+            child: new Text('$i', textDirection: TextDirection.ltr),
+          );
+        })
```

Comment:
nit: Flutter-style would be to add a comma after `}` and remove the newly introduced indentation.